.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: node_js
+sudo: required
+before_script:
```

Comment:
add `polymer lint` under the `before_script` section. Optionally, eslint, if you want to do that.